index.html

```diff
@@ -0,0 +1,76 @@
+        </svg>
+        <svg width="25" height="12" viewBox="0 0 25 12" fill="none" xmlns="http://www.w3.org/2000/svg" class="phone">
+          <rect opacity="0.35" x="0.5" y="0.833984" width="21" height="10.3333" rx="2.16667" stroke="black"/>
+          <path opacity="0.4" d="M23 4.00098V8.00098C23.8047 7.6622 24.328 6.87411 24.328 6.00098C24.328 5.12784 23.8047 4.33975 23 4.00098Z" fill="black"/>
+          <rect x="2" y="2.33398" width="18" height="7.33333" rx="1.33333" fill="black" />
+        </svg>
+      </div>
+    <div class="header2">
+      <div class="logo_welcome">
+        <p class="wlc">Welcome</p>
+      </div>
+      <div class ="union">
+        <svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" clip-rule="evenodd" d="M0.00012207 1C0.00012207 0.447715 0.447837 0 1.00012 0H17.0001C17.5524 0 18.0001 0.447715 18.0001 1C18.0001 1.55228 17.5524 2 17.0001 2H1.00012C0.447837 2 0.00012207 1.55228 0.00012207 1ZM0.00012207 6C0.00012207 5.44772 0.447837 5 1.00012 5L17.0001 5C17.5524 5 18.0001 5.44772 18.0001 6C18.0001 6.55229 17.5524 7 17.0001 7L1.00012 7C0.447837 7 0.00012207 6.55228 0.00012207 6ZM1.00012 10C0.447837 10 0.00012207 10.4477 0.00012207 11C0.00012207 11.5523 0.447837 12 1.00012 12H17.0001C17.5524 12 18.0001 11.5523 18.0001 11C18.0001 10.4477 17.5524 10 17.0001 10H1.00012Z" fill="#28352F"/>
+        </svg>
+      </div>
+    </div>
+  </header>
+    <div id="heading">
+      <div>
+        <h1 class="h1">Hey there. I’m <br> Freddy <br> <span class="sub_text">I’m a software developer</span></h1>
+        <p class="supporting">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <section class="social">
+        <nav>
```

Comment:
- [ ] kindly remove the `nav` tag that wrapping your social links , its not semantically correct , and <ul> tag should be enough 🙏